README.md

```diff
@@ -46,6 +46,14 @@ If you wish to regenerate it, you can delete this folder and:
  or
 2. cd into `pkg/didcomm/transport/http/` and run `go generate`
 
+##### Controller REST API specifications generation
+Controller REST API specifications are generated according to Open API 2.0 standards.
+
+It can be generated by running command `make generate-openapi-test`
+
+Generated spec can be found under 
+`build/rest/openapi/spec`
```

Comment:
It would be useful to also give instructions on how to run the openapi client with the generated specs as input.

Reply:
Will be added as part of #94